### ISSUE TITLE
Clear definition of "community" vs "informal" channels (for channel namespacing)

### ISSUE THREAD

**Issue by timmc** [Libera-Chat/libera-chat.github.io#47]
Currently it is not clear how to name a channel when migrating a topic channel from freenode, which had different policies. Specifically, it is not clear whether to use `#` or `##`; even if I register both "just in case", I don't know which I should be directing people to.

The guidelines at the moment tell me that community channels are [on-topic](https://libera.chat/chanreg#on-topic-projects) ("primarily free and open-source software projects, and other peer-directed projects") and "would otherwise be eligible for a project registration but do not have an official representation" (and gives examples of `#windows` and `#linux`.

To illustrate the problem (and I'm not asking for specific guidance on these channels) this is insufficient for making decisions about where freenode channels such as `##crypto` and `##infosec` should end up. These are software and tech related, and usually involve collaboration, open-source software, and other aspects of peer-directed interaction, but are not _projects_. At best, they touch on many different projects, or an aspect of a large number of projects.

Here's what I would like to see:

- Clear delineation of what is a "project"
- A short list of channels that would be on the _boundary_ between community and informal, with a brief explanation of why each is on the side that it falls on
- Just more examples in general, I guess :-)

**Comment by int-e** [comment]
The language about [communities](https://libera.chat/chanreg#community-registration) is probably more relevant. It allows (as one alternative) "topics that [...] are significantly relevant to our intended userbase", which is hard to delineate to my mind. Which side do subjects as databases, programming, cryptography, mathematics, physics, biology, history, english belong to?

**Comment by AbstractBeliefs** [comment]
@timmc is this addressed in https://libera.chat/policies/#channels ? 
There's also https://libera.chat/chanreg/#registering-a-channel , which is designed for those planning to register channels (and possibly a project).

I know it's been a while - it's likely possible that these didn't exist at the time. If they do hit what you had in mind, I'd like to bring it to your attention and possibly resolve the issue.

**Comment by mnordhoff** [comment]
No, the relevant parts of those documents haven't changed significantly since before this issue was opened.

**Comment by int-e** [comment]
I believe most of the workflow at https://libera.chat/chanreg/#how-to-register-as-a-community is newer than this ticket. Now at least it's clear whom to ask about eligibility for a community registration.

However, and that's the point of this ticket, the criteria for eligibility remain mysterious; it would be helpful to document them, or at least list some examples of successful and failed community registration that can be generalized.

**Comment by AbstractBeliefs** [comment]
I think part of the issue is that we don't want to try and overfit. Projects and communities are pretty varied and we're probably never going to be able to set out any kind of concise policy that gets the point across. That said, I'll see if we can word it better.

For example, I would say that infosec is welcome to # , but crypto would vary depending on the use: if it's cryptocurrency technology (for example the implementation and use of blockchain-backed ledgers) then in the round it could be #. If it were about the use, trading, and speculation of currencies, then it ought to be ##.

There's a wide grey area where the best option is to speak to the projects team, and where it's subjective. I'll reattack this over the weekend to see if I can get a new wording that conveys the broad intent.
